Lib9c/Action/ClaimStakeReward3.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Extensions;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    /// <summary>
+    /// Hard forked at https://github.com/planetarium/lib9c/pull/1371
+    /// </summary>
+    [ActionType(ActionTypeText)]
+    public class ClaimStakeReward3 : GameAction
+    {
+        public const long HardForkIndex = 5_599_601L;
```

Comment:
How do you think renaming this something like `ActionAvailableIndex`? This pull request contains `ActionUnavaliableException` so it can be more readable.